packages/fxa-auth-server/lib/senders/emails/templates/subscriptionSubsequentInvoiceDiscount/en.ftl

```diff
@@ -27,3 +27,10 @@ subscriptionSubsequentInvoiceDiscount-content-subtotal = Subtotal: { $invoiceSub
 # Variables:
 #  $invoiceDiscountAmount (String) - The amount of the discount of the subscription invoice, including currency, e.g. $2.00
 subscriptionSubsequentInvoiceDiscount-content-discount = Discount: -{ $invoiceDiscountAmount }
+# Variables
+#  $invoiceDiscountAmount (String) - The amount of the discount of the subscription invoice, including currency, e.g. $2.00
+subscriptionSubsequentInvoiceDiscount-content-discount-one-time = One Time Discount: -{ $invoiceDiscountAmount }
```

Comment:
I'd rather leave it because I think there is talks of having coupons available for upgraded subscriptions at some point and this will become relevant/needed.

Reply:
Ah yes good thinking!